src/TextArea/TextArea.less

```diff
@@ -44,9 +45,11 @@
   .border--s(transparent);
   .text--line-height-3;
   .padding--top--l;
+  padding-left: @size_4xs;
```

Comment:
This change, which isn't immediately apparent in the screenshots, is super subtle. It tweaks the left alignment of text in the component to match the left alignment of text in the `TextInput` component. (It was previously off by 1 pixel.)